release-notes/major9/09.20.0/migration.md

```diff
@@ -2,20 +2,17 @@
 
 This page describes how to update Valtimo from the previous version to the current.
 
-* **Breaking change 1/Deprecation 1**
-
-  1. **Step1**
-
-     Description
-  2. **Step2**
-
-     Description
-
-* **Breaking change 2/Deprecation 2**
-
-  1. **Step1**
-
-      Description
-  2. **Step2**
-
-      Description
+* **Added extra parameter to KeycloakService**
+
+  If your project has a custom implementation of the KeycloakService, your project might break due to a missing
+  parameter in the constructor called `keycloakClientName`. The solution in simple: Just add an empty string for
```

Comment:
This text feels very subjective. What's elegant and what isn't is up to debate. Maybe rephrase this entire section to something like:

* ** Added name parameter to KeycloakService**

  Custom implementations of the KeycloakService now require `keycloakClientName` as a parameter in the super constructor. Either set this value to an empty string (e.g. `super(properties, "")` or do the following:

  ```java
  public CustomKeycloakService(
      KeycloakSpringBootProperties properties,
      @Value("${valtimo.keycloak.client:}") String keycloakClientName
  ) {
      super(properties, keycloakClientName);
      this.properties = properties;
  }
  ```

Reply:
changed